### ISSUE TITLE
Integrate into the Prometheus web ui

### ISSUE THREAD

**Issue by SuperSandro2000** [prometheus/promlens#51]
It would be awesome if PromLens could just be another tab in Prometheus and those being fully integrated.

**Comment by juliusv** [comment]
Yep! That's pretty much the plan, see https://promlabs.com/blog/2022/10/25/promlabs-and-chronosphere-open-source-the-promlens-query-builder.

We just need to figure out a good way how to share the relevant bits of the code. There's still good reason for PromLens to additionally keep existing as a standalone tool, since it works not only with Prometheus, but also all other PromQL-compatible backends.

**Comment by Nexucis** [comment]
Hehe indeed :).

Like @juliusv said, we have some works to do before it can happen. 
Personally I would like to align the graphical chart between Prometheus and PromLens and then make some adjustment in PromLens so it is able to release an npm package that could be used in Prometheus to display PromLens.

Ideally, we would be able to include it in Thanos as well, since the UI are closed to what Prometheus is doing.
